src/tasks/defaults.js

```diff
@@ -131,6 +132,16 @@ module.exports = function(gulp, options, subtasks) {
     }));
 
     // Tasks that are just a collection of other tasks
+    
+    gulp.desc('connect:noReload', 'Start a server without livereload');
+    gulp.task('connect:noReload', subtasks.connect({livereload: false}));
+    
+    gulp.desc('connect:stop', 'Stop a running connect server allowing gulp to exit');
+    gulp.task('connect:stop', function (done) {
```

Comment:
You'll need to add dependencies to both of these new tasks:

```js
gulp.task('connect:noReload', getDeps(options, 'connect:noReload'), subtasks.connect({livereload: false}));
gulp.task('connect:stop', getDeps(options, 'connect:stop'), function (done) {...
```

and then add entries to the default taskTree in `src/gulpconfig.json` for them:

```json
"taskTree": {
    ...
    "connect:noReload": [],
    "connect:stop": [],
    ...
}
```

This will allow adding/overriding of dependencies via the taskTree configuration.